src/Compiler/CompilerUtils.cpp

```diff
@@ -411,6 +432,18 @@ static std::string getOptimizationLevelOption() {
   return "";
 }
 
+static std::string getXoptOption() {
+  return (Xopt != "") ? Xopt : std::string();
```

Comment:
So for the rest of the options, we return `--march=arch`, for example, but for these we just return the option. Is that going to get passed correctly to the underlying tools?

Reply:
yes that is intentional, given -Xopt --llvm_option_name  we will pass `--llvm_option_name" to the tool.